app/lua/luac_cross/luac.c

```diff
@@ -72,6 +73,7 @@ static void usage(const char* message)
  "  -f       output a flash image file\n"
  "  -a addr  generate an absolute, rather than position independent flash image file\n"
  "  -i       generate lookup combination master (default with option -f)\n" 
+ "  -m size  maximum LFS image in bytes\n"
  "  -p       parse only\n"
  "  -s       strip debug information\n"
  "  -v       show version information\n"
```

Comment:
Would it be possible to add integer/float information to the version command
Maybe in general stating that this is no normal luac.cross but LFS (and other) enhanced?

Reply:
Yes, we should say that this is a nodeMCU build with integer/float LFS/non-LFS support.